common/lib/xmodule/xmodule/js/spec/video/html5_video_spec.js

```diff
@@ -322,6 +323,22 @@
             });
         });
 
+        describe('non-hls encoding', function() {
+            beforeEach(function(done) {
+                state = jasmine.initializePlayer('video_html5.html');
```

Comment:
Should we also check for `jasmine.initializePlayerYouTube` or html5 player is enough